client/lib/analytics/fullstory.js

```diff
@@ -41,12 +47,16 @@ export function recordFullStoryEvent( name, _props ) {
 }
 
 function maybeAddFullStoryScript() {
+	const cookies = cookie.parse( document.cookie );
+	const isUS = 'US' === cookies.country_code;
```

Comment:
A couple of minor things here:

- `maybeAddFullStoryScript()` is fired quite a bit and we expect 99.9% of those calls to return early after seeing that `fullStoryScriptLoaded` is `true` already. I recommend that we not parse cookies on each of those calls, but rather parse cookies only when we actually need to do so — to improve performacne.
- Since this variable is used only once I suggest moving this to a single line down below.

```
'US' !== cookie.parse( document.cookie ).country_code ||
```